src/widget/form/profileform.cpp

```diff
@@ -247,10 +247,14 @@ void ProfileForm::onRenameClicked()
     Nexus& nexus = Nexus::getInstance();
     QString cur = nexus.getProfile()->getName();
     QString title = tr("Rename \"%1\"", "renaming a profile").arg(cur);
+
     do
     {
-        QString name = QInputDialog::getText(this, title, title+":");
-        if (name.isEmpty()) break;
+        QString name = QInputDialog::getText(0, title, title + ":");
```

Comment:
None of these works. The only way I found a few months ago was to remove parent.

![snapshot10](https://cloud.githubusercontent.com/assets/2544251/9099058/a7580806-3bc5-11e5-8c4b-f645031e78e1.png)